app/code/Magento/Theme/Block/Adminhtml/Wysiwyg/Files/Tree.php

```diff
@@ -57,7 +57,8 @@ public function getTreeLoaderUrl()
      */
     public function getTreeJson($data)
     {
-        return \Zend_Json::encode($data);
+        $serializer = new \Magento\Framework\Serialize\Serializer\Json;
```

Comment:
This is not correct. All dependencies shall be passed via constructor instead of direct instantiation.

Please fix all places.